inferentia/scripts/setup.sh

```diff
@@ -106,36 +127,32 @@ rm -rf build && mkdir build && cd build
 cmake -DTRITON_ENABLE_GPU=ON -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install ..
 make triton-python-backend-stub -j16
 
-# Install Neuron Pytorch
-# https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-intro/install-pytorch.html
-cd ${INFRENTIA_PATH}
-. /etc/os-release
-tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
-deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
-EOF
-wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB |  apt-key add -
-apt-get update && \ 
-    apt-get install -y       \
-        linux-headers-$(uname -r) \
-        aws-neuron-dkms           \
-        aws-neuron-runtime-base   \
-        aws-neuron-runtime        \
-        aws-neuron-tools 
-export PATH=/opt/aws/neuron/bin:$PATH
-
-# Install Neuron-pytorch (neuron-cc)
+# Set Pip repository  to point to the Neuron repository
+# since we need to use pip to update: 
+#  https://aws.amazon.com/blogs/developer/neuron-conda-packages-eol/
+pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
 conda config --env --add channels https://conda.repos.neuron.amazonaws.com
-conda install torch-neuron -y
 
-# Upgrade torch-neuron and install transformers
-# need to use pip to update: 
-# https://aws.amazon.com/blogs/developer/neuron-conda-packages-eol/
-pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
-pip install --upgrade torch-neuron torchvision "transformers==4.6.0"
+if [ $USE_TENSORFLOW -eq 0 ]
+then
+    conda install tensorflow-neuron pillow -y
+    #Update Neuron TensorFlow
+    pip install --upgrade tensorflow-neuron==1.15.5.* neuron-cc
+    # Update Neuron TensorBoard
+    pip install --upgrade tensorboard-plugin-neuron
+fi
+
+if [ $USE_PYTORCH -eq 0 ]
+then
+    conda install torch-neuron torchvision -y
+    # Upgrade torch-neuron and install transformers
+    pip install --upgrade torch-neuron neuron-cc[tensorflow] torchvision "transformers==4.6.0"
```

Comment:
Why `neuron-cc[tensorflow]` for `$USE_PYTORCH`?

Reply:
This is part of their installation instructions so I kept it

Reply:
This is strange... I am fine keeping it for now.